prerendering.bs

```diff
@@ -388,11 +346,11 @@ Every {{Document}} has an <dfn for="Document">activation start time</dfn>, which
 </div>
 
 <div algorithm="create a prerendering browsing context">
-  To <dfn export>create a prerendering browsing context</dfn> given a [=URL=] |startingURL|, a [=referrer policy=] |referrerPolicy|, and a {{Document}} |referrerDoc|:
```

Comment:
Similarly here, even if we're not going to key on it, it seems like we're going to want a policy plumbed through here.